h2o-algos/src/main/java/hex/gam/GAMModel.java

```diff
@@ -107,7 +107,7 @@ TwoDimTable genCoefficientMagTableMultinomial(String[] colHeaders, double[][] co
             "length: %d, coeffSigns length: %d", coeffNames.length, coeffMags.length, coeffSigns.length));
     
     int countIndex = 0;
-    for (int index = 0; index < nCoeff; index++) {
+    for (int index = 0; index < nCoeff - 1; index++) {
```

Comment:
Pavel:  I need to loop through all the coefficient names and to skip the intercept.  Sometimes the intercept can come first and sometimes it is last.  Ignore this for now since we are going to do Java inheriantance.

Reply:
This piece of code causes test failures.

Reply:
Then I know how to fix this.